iaso/api/profiles/profile_logs.py

```diff
@@ -0,0 +1,187 @@
+        serializer = NestedOrgUnitForListSerializer(org_units, many=True)
+        return serializer.data
+
+    def get_new_location(self, modification):
+        new_value = modification.new_value
+        if not new_value:
+            return []
+        org_unit_ids = new_value[0]["fields"].get("org_units", None)
+        if not org_unit_ids:
+            return []
+        org_units = OrgUnit.objects.filter(pk__in=org_unit_ids)
+        serializer = NestedOrgUnitForListSerializer(org_units, many=True)
+
+        return serializer.data
+
+    def get_fields_modified(self, modification):
+        diffs = modification.field_diffs()
+        if not modification.past_value:
+            return ["new_user_created"]
+        # password updated will always be in the added dict because it's never present on past value
+        password_updated = diffs["added"].get("password_updated", None)
+        if password_updated:
+            if password_updated.get("after", False):
+                diffs["added"]["password"] = password_updated
+            del diffs["added"]["password_updated"]
+
+        added = list(diffs["added"].keys())
+        removed = list(diffs["removed"].keys())
+        modified = list(diffs["modified"].keys())
+        result = list(set(removed + added + modified))
+        result.sort()
+        # filter user and account because these can't be modified
+        return [key for key in result if key != "user" and key != "account"]
+
+
+class ProfileLogRetrieveSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Modification
+        fields = ["id", "created_at", "user", "source", "new_value", "past_value", "object_id", "content_type"]
+
+
+class ProfileLogsViewset(ModelViewSet):
+    permission_classes = [HasPermission(permission.USERS_ADMIN)]
+    filter_backends = [
+        filters.OrderingFilter,
+        django_filters.rest_framework.DjangoFilterBackend,
+    ]
+    filterset_class = ProfileLogsListFilter
+    pagination_class = ProfileLogsListPagination
+    ordering_fields = ["created_at"]
+
+    http_method_names = ["get"]
+
+    def get_queryset(self):
+        order = self.request.query_params.get("order", "-created_at")
+        request_user = self.request.user
+
+        queryset = (
+            Modification.objects.select_related("user")
+            .filter(content_type__app_label="iaso")
+            .filter(content_type__model="profile")
+            .filter(user__iaso_profile__account=request_user.iaso_profile.account)
+        )
+
+        if "created_at" in order:
+            queryset = queryset.order_by(order)
+        if order == "modified_by":
+            queryset = queryset.order_by("user__username")
+        if order == "-modified_by":
+            queryset = queryset.order_by("-user__username")
+        if order == "user":
+            queryset = queryset.order_by("new_value__0__fields__user__username")
+        if order == "-user":
```

Comment:
```suggestion
        elif order == "modified_by":
            queryset = queryset.order_by("user__username")
        elif order == "-modified_by":
            queryset = queryset.order_by("-user__username")
        elif order == "user":
            queryset = queryset.order_by("new_value__0__fields__user__username")
        elif order == "-user":
```

I would use `elif` since you only get a single order crtierion